march_shared_classes/src/march_shared_classes/gait/subgait_graph.py

```diff
@@ -95,3 +95,9 @@ def __iter__(self):
                      for from_subgait, transitions in self._graph.items()
                      for to_subgait in transitions.values()
                      if len({from_subgait, to_subgait} & {self.START, self.END}) == 0])
+
+    def __eq__(self, other):
+        return isinstance(other, SubgaitGraph) and self._graph == other._graph
+
+    def __ne__(self, other):
+        return not self == other
```

Comment:
```suggestion
        return not self.__eq__(other)
```
In https://github.com/project-march/march/blob/develop/march_shared_classes/src/march_shared_classes/gait/setpoint.py we do it like this so maybe keep that format for consistency?